tool-labs/globalgroups/index.php

```diff
@@ -212,12 +224,16 @@
     "right-translate-sandboxmanage" => "Manage sandboxed users",
 
     // WikimediaMessages extension: https://github.com/wikimedia/mediawiki-extensions-WikimediaMessages/blob/master/i18n/wikimedia/en.json
-    "right-editeditorprotected" => "Edit pages protected as \"{{int:protect-level-editeditorprotected}}\"",
-    "right-editextendedsemiprotected" => "Edit pages protected as \"{{int:protect-level-editextendedsemiprotected}}\"",
+    "right-banner-protect" => "Protect translatable messages handled by the Translate extension and included in CentralNotice banners",
+    "right-docseditor" => "Edit documentation"
```

Comment:
There's a missing comma at the end of this line:

Reply:
Oopsie. Fixing in a sec. Thanks for the review.

Reply:
Done.